Chp7Lsn1/Chp7Lsn1/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Chp7Lsn1
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] arrayEmpty = new int[20];
+
+            int [] displayArray = FillArray(arrayEmpty);
+            DisplayArray(displayArray);
+        }
+
+        public static int[] FillArray(int[] arrayEmpty)
+        {
+            for(int i = 0; i < 20; i++)
+            {
+                arrayEmpty[i] = i * 5;
+            }
+
+            int[] arrayFull = arrayEmpty;
```

Comment:
This is an unnecessary step, but not a big deal.  Perhaps you just wanted "self-documenting variables?"